storage/range.go

```diff
@@ -433,7 +433,7 @@ func (r *Range) Desc() *proto.RangeDescriptor {
 // SetDesc atomically sets the range's descriptor. This method calls
 // processRangeDescriptorUpdate() to make the range manager handle the
 // descriptor update. Note that processRangeDescriptorUpdate()
-// acquires the metaLock.
+// acquires a lock on the range manager.
```

Comment:
So, `metaLock` is actually a member of `Range`, and it is explicitly locked in `AdminSplit`, `AdminMerge`, and `ChangeReplicas`.

Actually, we should remove this line, since the lock that `processRangeDescriptorUpdate` acquires when `rm` is a `Store` happens to acquire the `Store`'s `mu`, but that's not generally true for implementers of `rangeManager`.